assets/js/vue-apps/components/facet-select.vue

```diff
@@ -0,0 +1,34 @@
+<script>
+export default {
+    props: ['value', 'name', 'label', 'labelAny', 'options']
+};
+</script>
+
+<template>
+    <div class="u-margin-bottom-s" v-if="options.length > 0">
+        <label class="ff-label" :for="'field-dynamic-' + name">
```

Comment:
Is there a way to build this string once as a function rather than repeat it again in other lines/files?